fabric-fluids-v1/src/main/java/net/fabricmc/fabric/api/fluids/items/ItemSink.java

```diff
@@ -0,0 +1,26 @@
+package net.fabricmc.fabric.api.fluids.items;
+
+import net.minecraft.item.ItemStack;
+
+/**
+ * an interface through which items can be taken and removed from
+ * <p>
+ * implementors:
+ * {@link net.minecraft.entity.player.PlayerInventory}
+ */
+public interface ItemSink {
```

Comment:
This doesn't sound very fluid-like

Reply:
it's needed for buckets, couldn't think of a better way to implement it